src/darwin/Framework/CHIP/templates/partials/MTRCallbackBridge.zapt

```diff
@@ -110,5 +114,20 @@ void MTR{{> @partial-block}}SubscriptionBridge::OnSubscriptionEstablished(void *
          self->mEstablishedHandler = nil;
      }
 }
+
+void MTR{{> @partial-block}}SubscriptionBridge::OnDone()
```

Comment:
Can `KeepAliveOnCallback` and `OnDone` go into a base class instead of generating the same code for each instantiation?

Reply:
The class hierarchy here looks like this:
```
class MTROctetStringAttributeCallbackBridge : public MTRCallbackBridge<OctetStringAttributeCallback>
...
class MTROctetStringAttributeCallbackSubscriptionBridge : public MTROctetStringAttributeCallbackBridge
```

We don't really want OnDone and KeepAliveOnCallback to be public APIs except on the subscription bridge.  But I guess we can make them protected on `MTRCallbackBridge` and pull them in as public on the subscription bridge.  I'll do that.

Reply:
And we can think about pushing some members up from `MTRCallbackBridge` to non-templated `MTRCallbackBridgeBase`, I guess....  Filed https://github.com/project-chip/connectedhomeip/issues/23597